wordpress-mobile/WordPress-Android
# My Site Dashboard - Phase 2 - Remove Featured Flag

Closes #15658

This PR removes the feature flag for the `My Site Dashboard - Phase 2` feature. 

-----

To test:
1. Clear app data to make sure all feature flags are set to default.
2. Go to `Me` -> `App Settings` -> `Debug Settings`.
3. Notice that `my_site_dashboard_phase_2` feature flag is not present in `Remote features`.
4. Smoke test the app, specifically below areas:
    - Pull-to-refresh works as expected. 
       (Since `MySiteSourceManager` is modified to remove `My Site Dashboard - Phase 2` feature flag.)
    - Dashboard cards are shown whenever expected. 
       (Since `CardsBuilder` is modified to remove `My Site Dashboard - Phase 2` feature flag.)

-----

## Regression Notes
1. Potential unintended areas of impact

N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)

See `To test` section above.

3. What automated tests I added (or what prevented me from doing so)

N/A

PR submission checklist:
